app/assets/javascripts/client/home/StartAssessmentCtrl.js

```diff
@@ -0,0 +1,97 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient')
+      .controller('StartAssessmentCtrl', StartAssessmentCtrl);
+
+  StartAssessmentCtrl.$inject = [
+    '$scope',
+    '$timeout',
+    '$location',
+    '$modal',
+    'Assessment',
+    'SessionService'
+  ];
+
+  function StartAssessmentCtrl($scope, $timeout, $location, $modal, Assessment, SessionService) {
+    var vm = this;
+
+    vm.user = SessionService.getCurrentUser();
+    vm.district = vm.user.districts[0];
+
+    vm.alerts = [];
+    vm.assessment = {};
+
+    vm.hideModal = function() {
+      $('#startAssessment').modal('hide');
```

Comment:
I wonder if we should be using `$element` instead, otherwise the DOM element won't be found while running the test scenarios. I also don't see any tests for this method, do we need it?